packages/perseus/src/widgets/interactive-graphs/graphs/components/use-control-point.tsx

```diff
@@ -26,13 +25,18 @@ type Params = {
      * interactive figure on the graph.
      */
     sequenceNumber: number;
```

Comment:
How would you feel about making `sequenceNumber` optional, and then defaulting it to `1` below?

If this point has special aria label, it could be confusing to also require passing in a sequence number if it's not going to be used.